src/diy/smol-slime.md

```diff
@@ -1,640 +1,24 @@
-    * Serial Terminal (for sending commands to your Receiver/Trackers, [see alternatives](#accessing-the-serial-console))
-    * Toolchain Manager (for automatic setup of the toolchain for building firmware)
-        * 2.9.0 (Inside Toolchain Manager) Do not use a newer version!
-    * NOTE: Installing the Segger J-Link is not necessary for pre-defined boards.
-* <a href="https://code.visualstudio.com/download" target="_blank">VS Code</a> (For development purposes only)
-    * <a href="https://www.nordicsemi.com/Products/Development-tools/nRF-Connect-for-VS-Code" target="_blank">nRF Connect for VS Code</a> (Recommended)
-        * Install within VS Code extension tab, see the <a href="https://youtu.be/EAJdOqsL9m8" target="_blank">video tutorial</a>
-        * You may either install the <a href="https://marketplace.visualstudio.com/items?itemName=nordic-semiconductor.nrf-connect" target="_blank">extension itself</a> or the <a href="https://marketplace.visualstudio.com/items?itemName=nordic-semiconductor.nrf-connect-extension-pack" target="_blank">extension pack</a> for additional development tools
-    * You may also set up a manual build environment in VS Code as the extension is known to fail on some certain Linux distributions.
-* <a href="https://slimevr.dev/download" target="_blank">SlimeVR Server</a>
-    * 0.13.2 or later version
-
-You only need the following if you are using precompiled firmware:
-* <a href="https://www.nordicsemi.com/Products/Development-tools/nRF-Connect-for-Desktop" target="_blank">nRF Connect for Desktop</a> (Programmer) for flashing Nordic or eByte Dongles only
-* <a href="https://www.nordicsemi.com/Products/Development-tools/nRF-Connect-for-Desktop" target="_blank">nRF Connect for Desktop</a> (Serial Terminal) for sending commands to your Receiver/Trackers, [see alternatives](#accessing-the-serial-console)
-* <a href="https://slimevr.dev/download" target="_blank">SlimeVR Server</a>
-    * 0.13.2 or later version
-
-## Firmware
-```admonish important
-The recommended method of getting the firmware is in the [pre-compiled section](#pre-compiled-firmware-for-default-pins) if you don't need custom config or pin defines.
-```
-
-### Cloning Repositories
-1. Open the Command Prompt by typing ```cmd``` in the Start Menu.
-1. Navigate to the directory where you want to clone the repositories.. (Type "cd" followed by a space and then the full path to the desired folder or drive.)
-1. Cloning SlimeNRF Receiver Repository.
-```
-git clone --single-branch --recurse-submodules -b master https://github.com/SlimeVR/SlimeVR-Tracker-nRF-Receiver.git
-```
-4. Cloning the SlimeNRF Tracker Repository.
-```
-git clone --single-branch --recurse-submodules -b master https://github.com/SlimeVR/SlimeVR-Tracker-nRF.git
+If you use one of existing case designs you likely don't have to worry about firmware and just pick one of prebuilt firmware, but you also can built your own. Page about this: [Smol Firmware](smol-slimes/smol-firmware.md).
```

Comment:
please reword this

Reply:
"Community designs will usually work with a prebuilt firmware. You can find firmware at \[Pre-Compiled Firmware](smol-slimes/smol-firmware.md#pre-compiled-firmware-for-default-pins). If you want to build the firmware yourself, see \[Compiling Firmware](smol-slimes/smol-firmware.md#compiling-firmware)."